tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -144,8 +144,12 @@ try {
       sources.push_back(std::move(source));
    }
 
+   // Always write the RNTuple and the file with the same compression.
+   outFile->SetCompressionSettings(compression);
+
    RNTupleWriteOptions writeOpts;
    assert(compression != kUnknownCompressionSettings);
+   writeOpts.SetUseBufferedWrite(false);
```

Comment:
This should go to the previous commit... On the other hand it probably doesn't do anything since we are not using `RPageSinkBuf` in the first place, but construct a `RPageSinkFile` manually, so might as well just drop it (in a separate commit)